contracts/cw-core/src/state.rs

```diff
@@ -20,6 +20,9 @@ pub struct Config {
     /// If true the contract will automatically add received cw721
     /// tokens to its treasury.
     pub automatically_add_cw721s: bool,
+    /// The URI for the DAO as defined by the DAOstar standard
+    /// https://daostar.one/EIP
+    pub dao_uri: Option<String>,
```

Comment:
If we are modifying existing state we will have to handle this in `migrate`

Reply:
Oh, yes! Want to add a commit and make sure it's tested?

Reply:
sure will do